contents/blog/the-posthog-array-1-28-0.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2021-09-14
+title: Array 1.28.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: release-notes
+featuredImage: ../images/blog/array/1-28-0.png
+excerpt: PostHog 1.28.0 has launched! Excerpt here
+---
+
+PostHog 1.28.0 has launched! 
+
+Summary here
+
+### Community MVP 🏆
+
+Thanks to all our community members for helping move PostHog forward! This release cycle's Community MVP goes to... [tbd](https://github.com/posthog)!
+
+Description here
+
+### Help us improve PostHog
+
+We’re working hard to improve the PostHog experience and would love to talk to you! Please join one of our Product, Engineering or Marketing team members in a quick 30-min call to help us understand how to improve. Schedule directly [on Calendly](https://calendly.com/posthog-feedback).
+
+As a small thank you for your time, we're giving away some awesome [PostHog merch](https://merch.posthog.com)!
+
+<ArrayCTA />
+
+## PostHog 1.28.0 Release Notes
+
+> If you're self-hosting and want to upgrade for a better experience and new features, remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
+
+**In this release:**
+
+- [Significantly revamped performance](#significantly-revamped-performance).
+
+### Significantly revamped performance
+
+Description here.
+
+### Feature 2
+Description here.
+
+
+### Improvements & fixes
+
+- List here
```

Comment:
```suggestion
- On open-source clickhouse, we now automatically create columns for event and person properties to speed up queries during weekends. This can speed up your slowest queries 2-25x.
```

I should write up some docs regarding this as well :sweat_smile: bit late though.

Reply:
Thanks @macobo!